torch/export/exported_program.py

```diff
@@ -223,10 +223,10 @@ def assert_valid_to_preserve(op_overload):
         if torch._C.DispatchKey.CompositeImplicitAutograd in op_overload.py_kernels:
             del op_overload.py_kernels[torch._C.DispatchKey.CompositeImplicitAutograd]
 
-        def _(*args, **kwargs):
-            return NotImplemented
+            def _(*args, **kwargs):
```

Comment:
This means we are only doing this override if there is a python CompositeImplicitAutograd kernel registered but most of the CIA ops decomp implementation exist in C++ i think? So i think this change will be unsafe.

Reply:
yeah seems like it's causing a bunch of failures :P